README.md

```diff
@@ -22,6 +22,35 @@ A detailed tutorial is provided on our [website](https://federatedscope.io/).
 - [05-25-2022] Our paper [FederatedScope-GNN](https://arxiv.org/abs/2204.05562) has been accepted by KDD'2022!
 - [05-06-2022] We release FederatedScope v0.1.0! 
 
+## Code Structure
+```
+FederatedScope
+├── federatedscope
+│   ├── core           
+│   |   ├── worker               # Behaviors of participants (i.e., server and clients)
+│   |   ├── configs              # Customizable configurations
+│   |   ├── trainer              # Detail of local training
+│   |   ├── monitors             # The monitor module for logging and demonstrating  
+│   |   ├── aggregator.py        # Detail of federated aggregation
+│   |   ├── communication.py     # Implementation of communication among participants   
+│   |   ├── fed_runner.py        # The runner for building and running an FL course
+│   |   ├── ... ..
+│   ├── cv                       # Federated learning in CV        
+│   ├── nlp                      # Federated learning in NLP          
+│   ├── gfl                      # Graph federated learning          
+│   ├── autotune                 # Auto-tunning for federated learning         
+│   ├── main.py           
+│   ├── ... ...          
+├── scripts                      # Scripts for reproducing existing algorithms
+├── benchmark                    # We release several benchmarks for convenient and fair comparisons
+├── doc                          # For automatic documentation
+├── enviornment                  # Installation requirements and provided docker files
+├── materials                    # Materials of related topics (e.g., paper lists)
```

Comment:
`materials` may be too abstract. At the first glance of the directory `materials`, it's hard to think of the paper list.

Reply:
I add the subfiles of materials here